bibliopixel/project/alias_lists.py

```diff
@@ -7,18 +7,17 @@
 
 BUILTIN_ALIASES = {
     # drivers
-    'apa102': 'bibliopixel.drivers.API.APA102.APA102',
```

Comment:
Whaaaat?! How the hell did we miss that one! Well, other than the fact that I always use `spi` with `ledtype`